### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by philippgualdi**
Update german translations and add frontend translations

### CODE REVIEW

**Review by utnapischtim — APPROVED** [review]

File: invenio_records_marc21/translations/de/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -8,77 +8,109 @@ msgid ""
...
 msgid ""
 "Connection to Services refused.\n"
 "Maybe you missed to start the services.\n"
 "Try to do a invenio-cli services setup -f"
 msgstr ""
 
-#: invenio_records_marc21/services/config.py:42
+#: invenio_records_marc21/errors.py:56
+msgid "User has not the permissions."
+msgstr "Benutzer hat nicht die erforlderichen Berechtigung."
```
Comment (utnapischtim) [inline invenio_records_marc21/translations/de/LC_MESSAGES/messages.po:53]: Typo

File: invenio_records_marc21/translations/de/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -8,77 +8,109 @@ msgid ""
...
+#: invenio_records_marc21/services/config.py:52
 msgid "Public"
-msgstr ""
+msgstr "Öffentlich"
 
-#: invenio_records_marc21/services/config.py:45
+#: invenio_records_marc21/services/config.py:53
 msgid "Embargoed"
-msgstr ""
+msgstr "Gesperrt"
```
Comment (utnapischtim) [inline invenio_records_marc21/translations/de/LC_MESSAGES/messages.po:66]: "Gesperrt" is not the correct translation. Where is this used? "Embargo" is also used in German.

File: invenio_records_marc21/translations/de/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -8,77 +8,109 @@ msgid ""
...
 
-#: invenio_records_marc21/services/config.py:53
+#: invenio_records_marc21/services/config.py:61
 msgid "Unpublished"
-msgstr ""
+msgstr "Unveröffentlicht"
+
+#: invenio_records_marc21/services/schemas/__init__.py:34
+msgid "Invalid persistent identifier scheme."
+msgstr "Ungültiges Schema für permanente Kennung."
```
Comment (utnapischtim) [inline invenio_records_marc21/translations/de/LC_MESSAGES/messages.po:86]: This german translation seams not right?

File: invenio_records_marc21/translations/de/LC_MESSAGES/messages.po (hunk 2 above)
Comment (philippgualdi) [reply invenio_records_marc21/translations/de/LC_MESSAGES/messages.po:66]: In the Invenio environment this translations is used in german for embargoed records.

File: invenio_records_marc21/translations/de/LC_MESSAGES/messages.po (hunk 3 above)
Comment (philippgualdi) [reply invenio_records_marc21/translations/de/LC_MESSAGES/messages.po:86]: Same here

**Pull request merged**